src/NovelRT/CMakeLists.txt

```diff
@@ -161,11 +161,38 @@ if(WIN32)
       Winmm
   )
 
+  set(CMAKE_INSTALL_PREFIX "C:/Program Files/NovelRT" CACHE PATH "..." FORCE)
+
   # I hate CMake I hate it I hate it I hate it
   # (work around for CMAKE REFUSING TO COPY NECESSARY FILES FOR DEBUGGING MAKING IT IMPOSSIBLE TO DEBUG)
-  add_custom_command(TARGET Engine POST_BUILD COMMAND ${CMAKE_COMMAND} -E copy_directory
-      ${CMAKE_BINARY_DIR}/thirdparty
-      $<TARGET_FILE_DIR:Engine>)
-  file(GLOB libs RELATIVE ${CMAKE_BINARY_DIR}/thirdparty ${CMAKE_BINARY_DIR}/thirdparty/*.dll)
+  #add_custom_command(TARGET Engine POST_BUILD COMMAND ${CMAKE_COMMAND} -E copy_directory
+  #    ${CMAKE_BINARY_DIR}/thirdparty
+  #    $<TARGET_FILE_DIR:Engine>)
+  #file(GLOB libs RELATIVE ${CMAKE_BINARY_DIR}/thirdparty ${CMAKE_BINARY_DIR}/thirdparty/*.dll)
```

Comment:
Note to self - this breaks debugging on Windows...
@FiniteReality can we even remove this line? Or would this be subject to rework using RUNTIME_DEPENDENCIES like you mentioned? (Mainly because Linux does not appear to be an issue, only Windows)

Reply:
I think using RUNTIME_DEPENDENCIES should fix it? Looking closer, it seems like that's a 3.21 feature though so we might not be able to support it.

Reply:
I just saw an article about that - we'd have to bump the version up but it may work...not sure if that would be a bad thing or not 🤔

Reply:
@RubyNova - assuming you meant to comment here about debugging being broken, it currently uses the lines that are commented out (as it was called a hack lol). When commented, debugging breaks because the DLLs aren't copied to the engine path anymore